src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CaseNotesNotificationsServiceTest.kt

```diff
@@ -44,9 +44,9 @@ internal class CaseNotesNotificationsServiceTest {
   @Test
   fun `both PPs (responsible officer) and SPs (assignee) get notifications for a sent case note as long as they didn't send it`() {
 
-    whenever(hmppsAuthService.getUserDetail(any<AuthUser>())).thenReturn(UserDetail("sp", "sp@provider.co.uk"))
+    whenever(hmppsAuthService.getUserDetail(any<AuthUser>())).thenReturn(UserDetail("sp", "sp@provider.co.uk", "last"))
```

Comment:
_impact of change in other files_

This class changed but its production code equivalent did not -- should we consider extracting these test objects into a constant/factory?

This applies to all tests that only changed in `thenReturn` data and no production code changes

Reply:
yeh this is a fair point. we do do this elsewhere with our factory class pattern. i guess in the past it was thought we didn't need a factory for such a skinny object, but as you say, it does make sense for this reason.